apps/cdk/src/app/frontend-production.ts

```diff
@@ -0,0 +1,16 @@
+import { App } from 'aws-cdk-lib';
+
+import { waitForStackIdle } from '../lib/wait-for-stack-idle';
+import { FrontendStack } from '../stacks/frontend';
+
+async function main() {
+    const stackName = 'antalmanac-frontend-production';
+
+    await waitForStackIdle(stackName);
+
+    const app = new App({ autoSynth: true });
+
+    new FrontendStack(app, stackName, { env: { region: 'us-east-1' } });
```

Comment:
This should be US West 1, I think. Generally, staging is US East, and production is US West because the former has more features, but the latter is closer to users.